search/Bfs.cc

```diff
@@ -363,42 +359,45 @@ BfsFwdIterator::incrLevel(Level &level) const
 
 bool
 BfsFwdIterator::levelLessOrEqual(Level level1,
-				 Level level2) const
+                                 Level level2) const
 {
   return level1 <= level2;
 }
 
 bool
 BfsFwdIterator::levelLess(Level level1,
-			  Level level2) const
+                          Level level2) const
 {
   return level1 < level2;
 }
 
 void
 BfsFwdIterator::enqueueAdjacentVertices(Vertex *vertex,
-					SearchPred *search_pred,
-					Level to_level)
+                                        SearchPred *search_pred,
+                                        Level to_level)
 {
   if (search_pred->searchFrom(vertex)) {
     VertexOutEdgeIterator edge_iter(vertex, graph_);
     while (edge_iter.hasNext()) {
       Edge *edge = edge_iter.next();
       Vertex *to_vertex = edge->to(graph_);
-      if (to_vertex->level() <= to_level
-	  && search_pred->searchThru(edge)
-	  && search_pred->searchTo(to_vertex))
-	enqueue(to_vertex);
+      if (to_vertex->level() <= to_level && search_pred->searchThru(edge)
+          && search_pred->searchTo(to_vertex))
+        enqueue(to_vertex);
     }
   }
 }
 
 ////////////////////////////////////////////////////////////////
 
 BfsBkwdIterator::BfsBkwdIterator(BfsIndex bfs_index,
-				 SearchPred *search_pred,
-				 StaState *sta) :
-  BfsIterator(bfs_index, level_max, 0, search_pred, sta)
+                                 SearchPred *search_pred,
+                                 StaState *sta) :
+  BfsIterator(bfs_index,
```

Comment:
I could live with this but I don't understand why these args are on separate lines when other function calls are broken by line width.